gqcp/include/QCMethod/HF/UHF/UHFSCFEnvironment.hpp

```diff
@@ -78,16 +75,15 @@ class UHFSCFEnvironment {
     /**
      *  A constructor that initializes the environment with initial guesses for the alpha and beta coefficient matrices.
      * 
-     *  @param N_alpha                  the number of alpha electrons (the number of occupied alpha-spin-orbitals)
-     *  @param N_beta                   the number of beta electrons (the number of occupied beta-spin-orbitals)
-     *  @param sq_hamiltonian           the Hamiltonian expressed in the scalar (AO) basis
-     *  @param S                        the overlap matrix (of the scalar (AO) basis)
+     *  @param N_alpha                  The number of alpha electrons (the number of occupied alpha-spin-orbitals).
+     *  @param N_beta                   The number of beta electrons (the number of occupied beta-spin-orbitals).
+     *  @param sq_hamiltonian           The Hamiltonian expressed in the scalar (AO) basis.
+     *  @param S                        The overlap matrix (of the scalar (AO) basis).
      *  @param C_alpha_initial          The initial coefficient matrix for the alpha spin-orbitals.
      *  @param C_beta_initial           The initial coefficient matrix for the beta spin-orbitals.
      */
-    UHFSCFEnvironment(const size_t N_alpha, const size_t N_beta, const USQHamiltonian<Scalar>& sq_hamiltonian, const SquareMatrix<Scalar>& S, const UTransformationComponent<Scalar>& C_alpha_initial, const UTransformationComponent<Scalar>& C_beta_initial) :
-        N_alpha {N_alpha},
-        N_beta {N_beta},
+    UHFSCFEnvironment(const size_t N_alpha, const size_t N_beta, const USQHamiltonian<Scalar>& sq_hamiltonian, const ScalarUSQOneElectronOperator<Scalar>& S, const UTransformationComponent<Scalar>& C_alpha_initial, const UTransformationComponent<Scalar>& C_beta_initial) :
```

Comment:
Perhaps we could also 
```suggestion
    UHFSCFEnvironment(const size_t N_alpha, const size_t N_beta, const USQHamiltonian<Scalar>& sq_hamiltonian, const ScalarUSQOneElectronOperator<Scalar>& S, const UTransformation<Scalar>& C_initial) :
```

Perhaps we could also change from the 2 arguments to the 1 encapsulating argument? (This also applies to the Python bindings.)